exo-sync/bin/setup

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-npm install --loglevel error --depth 0
+yarn install
```

Comment:
I've found that yarn developers strongly recommend against installing through npm. should we install through npm anyway? Or how about install it locally instead?